README.md

```diff
@@ -146,18 +148,20 @@ If you are looking for the codebase of the current production version of GATK, p
             --num-executors 5 --executor-cores 2 --executor-memory 4g \
             --conf spark.yarn.executor.memoryOverhead=600
         ```
-
+    * When using Dataproc you can access Web interface from YARN, Hadoop and HDFS. Follow [those instructions] (https://cloud.google.com/dataproc/cluster-web-interfaces) to create an SSH tunnel and connect with your browser.
```

Comment:
"access Web interface from" -> "access the web interfaces for"
"those instructions" -> "these instructions"